Basic/cordapp-example/workflows-java/src/main/resources/migration/iou.changelog-v1.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="R3.Corda" id="create_iou_state">
+        <createTable tableName="iou_states">
+            <column name="output_index" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="transaction_id" type="NVARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="value" type="int"/>
+            <column name="lender" type="NVARCHAR(64)"/>
+            <column name="borrower" type="NVARCHAR(64)"/>
+            <column name="linear_id" type="uuid"/>
```

Comment:
I believe this needs to beof type  `VARCHAR(64)` when using `uuid-char` in the type annotation.